docs/web-chat/api.mdx

```diff
@@ -19,7 +19,7 @@ import {
   Step,
 } from "@site/src/components/forDocs";
 
-# Helpshift APIs
+# Helpshift Widget APIs
```

Comment:
@hs-umang This is wrong. Web chat doc should not be updated

Reply:
Ack

Reply:
Ack

Reply:
Update in the next commit